logproof/src/linear_relation.rs

```diff
@@ -1501,7 +1486,7 @@ mod test {
         let mut vals = vec![];
 
         for i in 0..257 {
-            let ring_val = Zr::try_from(i).unwrap();
+            let ring_val = Zr::from(i);
```

Comment:
I forget if I provided a from implementation. What does it do if i > p?

Reply:
It [doesn't seem to check that](https://github.com/Sunscreen-tech/Sunscreen/blob/5cccd88ac0ee4ab53287a4ee0a0fed2126abd2b4/sunscreen_math/src/ring/mod.rs#L533-L537).

Also just to be clear, these `try_from -> from` warnings are _because_ there are `From` implementations and _not_ `TryFrom` implementations (offering both would conflict).